CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 ## Unreleased
+
+## v0.15.0
+Improvements:
+* Updated dependencies: [GH-85](https://github.com/hashicorp/vault-plugin-database-elasticsearch/pull/85)
```

Comment:
I think we should keep with precedent and list out the dependency upgrades (packages and new versions)